libcontainer/container_linux.go

```diff
@@ -488,9 +488,10 @@ func (c *Container) commandTemplate(p *Process, childInitPipe *os.File, childLog
 
 	cmd.ExtraFiles = append(cmd.ExtraFiles, childLogPipe)
 	cmd.Env = append(cmd.Env,
-		"_LIBCONTAINER_LOGPIPE="+strconv.Itoa(stdioFdCount+len(cmd.ExtraFiles)-1),
-		"_LIBCONTAINER_LOGLEVEL="+p.LogLevel,
-	)
+		"_LIBCONTAINER_LOGPIPE="+strconv.Itoa(stdioFdCount+len(cmd.ExtraFiles)-1))
+	if p.LogLevel != "" {
```

Comment:
Not new, but `LogLevel` is weird; it's a string, but it's not storing a named level, but a string representation of the numeric value (this code is converting the `logrus.Level` to an integer -> string, instead of taking its name; https://github.com/opencontainers/runc/blob/5f5b35dad46455dddd007d95b12ae78081377a02/utils_linux.go#L217

Reply:
This was done deliberately in commit 6c4a3b13d132aa460e610cf2c972e9ed71e4a42d (part of #3201) as a preparation for f1b703fc45dd61429634e9d448206c9b75d87147 (which would be much more complicated otherwise).

I guess what's missing from that PR is a docstring saying this is a string representation of a numeric log level. I'll open a PR to add it.

Reply:
https://github.com/opencontainers/runc/pull/3852